doc/nvim-tree-lua.txt

```diff
@@ -442,6 +442,14 @@ Following is the default configuration. See |nvim-tree-opts| for details.
             none = " ",
           },
         },
+        size = {
+          enable = false,
+          width_cutoff = 18,
+          column_width = 10,
+          show_folder_size = false,
+          format_unit = "double",
+          noshow_folder_size_glyph = "•",
```

Comment:
I'm always nervous about too many options however I predict that users will want:
`precision = 2`

I know that I'll be setting it to 0.

Reply:
yes, as I'm using I feel the 2 precision is just unnecessary. I'll get on it.